Source: Anders Ossowicki
Upstream: no
Reason: bacus.pt doesn't reply. Fixes test-phase
--- t/handle/external/10_destroy.t.orig	2007-07-03 13:53:43.000000000 +0000
+++ t/handle/external/10_destroy.t	2008-05-05 20:24:58.000000000 +0000
@@ -6,7 +6,7 @@
 
 my @uris = qw(
         debianforum.de
-        bacus.pt
+        exherbo.org
         perldition.org
 );
 
